dask/bytes/tests/test_compression.py

```diff
@@ -0,0 +1,39 @@
+from io import BytesIO
+
+import pytest
+from dask.bytes.compression import compress, decompress, files
+
+def test_compression():
+    assert set(compress) == set(decompress)
+
+    a = b'Hello, world!'
+    for k in compress:
+        comp = compress[k]
+        decomp = decompress[k]
+        b = comp(a)
+        c = decomp(b)
+        assert a == c
+        if k is not None:
+            assert a != b
```

Comment:
Yeah, that makes sense.  Same argument as above here.  In practice this is always true and the likelihood of my screwing up is higher than the likelihood of pass through compression.